docs/properties.md

```diff
@@ -352,6 +353,31 @@ For both implicit and explicit values, the format is an integer.
 }
 ```
 
+### MaterialColors
+
+For both implicit and explicit values, the type is an object that looks like this:
+
+```json
+{
+	"$properties": {
+		"MaterialColors": {
```

Comment:
We usually have `"ImplicitExample"` and `"ExplicitExample"` to make it clear which is which. Here they are both called `"MaterialColors"` which breaks our norm and makes it a little confusing (especially since this is technically invalid as it contains duplicate keys).